apps/website/tests/households.test.ts

```diff
@@ -23,6 +23,58 @@ test('User can create household', async ({ page }) => {
     .filter({ hasText: /^Add$/ })
     .getByRole('button')
     .click();
-  // await page.getByRole('button', { name: 'Add' }).click();
   await expect(page.getByText('New household')).toBeVisible();
+  await page.getByLabel('Household Name').fill('Household 1');
+  await page.getByRole('dialog').getByRole('button', { name: 'Add' }).click();
+  await expect(page.getByRole('dialog')).not.toBeVisible();
+  expect(
+    page.getByTestId('sidebar-household').getByText('Household 1'),
+  ).toBeVisible();
+});
+
+test('User can view household details', async ({ page }) => {
+  await page.goto('/');
+  await page.goto('/dashboard/household');
+  await login(page);
+
+  await page.getByTestId('sidebar-household').getByText('Household 1').click();
+  await expect(
+    page.getByRole('heading', { name: 'Household 1' }),
+  ).toBeInViewport();
+});
+
+test('User can edit', async ({ page }) => {
+  await page.goto('/');
+  await page.goto('/dashboard/household');
+  await login(page);
+
+  await page.getByTestId('sidebar-household').getByText('Household 1').click();
+
+  await page.getByRole('button', { name: 'Edit' }).click();
+  await page.getByLabel('Household Name').clear();
+  await page.getByLabel('Household Name').fill('Edited Household 1');
+  await page.getByRole('dialog').getByRole('button', { name: 'Save' }).click();
+  await expect(
+    page.getByTestId('sidebar-household').getByText('Edited Household 1'),
+  ).toBeVisible();
+});
+
+test('User can delete household', async ({ page }) => {
+  await page.goto('/');
```

Comment:
thing to figure out in the future: why do I need to go to the main route first before I can consistently login